Sources/Sentry/include/SentrySdkInfo.h

```diff
@@ -0,0 +1,38 @@
+#import <Foundation/Foundation.h>
+
+#import "SentrySerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Describes the Sentry SDK and its configuration used to capture and transmit an event.
+ *
+ * Both name and version are required.
+ *
+ * For more info checkout: https://develop.sentry.dev/sdk/event-payloads/sdk/
+ */
+@interface SentrySdkInfo : NSObject <SentrySerializable>
+SENTRY_NO_INIT
+
+/**
+ * The name of the SDK.
+ *
+ * Examples: sentry.cocoa, sentry.python, ...
```

Comment:
L: What about we suggest a possible variation that would be relevant to this code base?
If we created a [vapor](https://github.com/vapor/vapor) integration which would be its own SDK on top of sentry.cocoa, it would be named `sentry.cocoa.vapor`:

```suggestion
 * Examples: sentry.cocoa, sentry.cocoa.vapor, ...
```